### ISSUE TITLE
Update JEC tag

### ISSUE THREAD

**Issue by gkasieczka** [vhbb/cmssw#643]
New recommendation will be V3. Should become available next days.

https://indico.cern.ch/event/607325/contributions/2447317/attachments/1400235/2136634/20170123_JERC_status.pdf

**Comment by veelken** [comment]
Hi Gregor,

did you succeed in integrating the new JEC yet ?
I don't see a corresponding PR, but maybe it is merged already ?

**Comment by gkasieczka** [comment]
Yes, it was already in (and caused the submission hiccup Andrea spotted and
fixed today)

**Comment by veelken** [comment]
Hi Gregor,

thank you very much for the information.
